niceman/utils.py

```diff
@@ -986,7 +988,47 @@ def get_cmd_batch_len(arg_list, cmd_len):
     return max((_MAX_LEN_CMDLINE - cmd_len) // (max_len + 1), 1)
 
 
-def execute_command_batch(session, command, args, expected_exceptions=None):
+def join_sequence_of_dicts(seq):
+    """
+    Joins a sequence of dicts into a single dict, trows RuntimeError if dupe
```

Comment:
```
From WordNet (r) 3.0 (2006) [wn]:

  dupe
      n 1: a person who is tricked or swindled [syn: {victim}, {dupe}]
      v 1: fool or hoax; 
```

Reply:
"dupe" = short for "duplicate" -- The joys of trying to fit things in 80 characters :)

Reply:
The Oxford English dictionary does recognize "dupe" as short for "duplicate": https://en.oxforddictionaries.com/definition/dupe#dupe-2

Reply:
Another option: End the summary sentence at dict and then add

```
Raises
------
RuntimeError if a duplicate key is encountered.
```

to the end of the docstring.

Reply:
@kyleam for the win!  Updated...